### ISSUE TITLE
Nightowl II research doesn't depend on Nightown I

### ISSUE THREAD

**Issue by MatthiasMann** [ldtteam/minecolonies#5947]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.15.2
- MineColonies Version: 0.13.358-alpha
- Structurize Version: 0.14.94-alpha
- Related Mods and their Versions:

## Expected behavior
Nightowl II is an improvement of Nightowl I

## Actual behavior
I can research Nightowl II before researching Nightowl I
![2020-10-18_14 54 30](https://user-images.githubusercontent.com/6777029/96368104-435b9d00-1152-11eb-9f43-4d11ac541115.png)

## Steps to reproduce the issue
1. Have the Festival research
2. Both options are available in parallel

### Logs
- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Aduis1210** [comment]
Noticed as well

**Comment by someaddons** [comment]
that one is actually intended

**Comment by ravenbuilder934** [comment]
It is? Why? Then you could research nightowl II before nightowl I

**Comment by Raycoms** [comment]
They are accumulative (total of 3h when researched both)
